### ISSUE TITLE
DateTime

### ISSUE THREAD

**Issue by PeterHaro** [damienbod/AspNetCoreCsvImportExport#21]
Is there any, easy, way to serve the CSV with a custom datetime formatting?

**Comment by arivera12** [comment]
I don't think this is supported but you may try changing the thread culture or creating a custom model with the desired format.
